specification/cat/_types/CatBase.ts

```diff
@@ -28,3 +28,91 @@ import { RequestBase } from '@_types/Base'
 export class CatRequestBase
   extends RequestBase
   implements CommonCatQueryParameters {}
+
+export enum CatDfaColumn {
+  /**
+   * Contains messages relating to the selection of a node.
+   * @aliases ae
+   */
+  assignment_explanation = 0,
```

Comment:
You can drop all the ` = <number>`, it is not needed and just `assignment_explanation,` is fine.

There is still a lot of occurrences of "numbered enumerations" in the TypeScript files that were output by the script that created the initial version of the spec from the Nest client code. We have to clean things up and remove them all, and can happily avoid adding them in new specifications.